Ships as Raw

```diff
@@ -0,0 +1,860 @@
+
+Name|Gladiator I-class Star Destroyer
+Type|Gladiator
+Keywords|0
+Faction|Imperial
+Size|Small
+Points|56
+Hull|5
+defenseTokens|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Speed1|2
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|3
+SideShields|2
+RearShields|1
+FrontArmament|RRKK
+SideArmament|KKKK
+RearArmament|RK
+AntiSquadron|B
+Upgrade|Title Officers Weapon Team Support Ordance
+FrontArcOffset|8.7
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|7.5
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|33
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|37
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Gladiator II-class Star Destroyer
+Type|Gladiator
+Keywords|0
+Faction|Imperial 
+Size|Small
+Points|62
+Hull|5
+defenseTokens|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Speed1|2
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|3
+SideShields|2
+RearShields|1
+FrontArmament|RRKK
+SideArmament|KKKR
+RearArmament|RK
+AntiSquadron|BB
+Upgrade|Title Officers Weapon Team Support Ordance

```

Comment:
I'd prefer Officer over Officers, since its one officer for the slot, and that's the name given in the wiki, and maybe even the rulebook.  but I'm willing to not push to hard on that.